workers/data_refinery_workers/processors/transcriptome_index.py

```diff
@@ -66,8 +66,13 @@ def _compute_paths(job_context: Dict) -> str:
     job_context["rsem_index_dir"] = job_context["work_dir"] + "rsem_index/"
     os.makedirs(job_context["rsem_index_dir"], exist_ok=True)
 
-    # I think this is a bit sketchy.
-    job_context["organism_name"] = job_context["base_file_path"].split("/")[-1]
+    # We don't actually associated the organism with the job, so we
+    # have to determine the organism name from the file
+    # path. Sometimes these files have strain information appended to
+    # them, so we want to make sure we only use the first two parts of
+    # a name like Candida_albicans_sc5314_gca_000784635
+    organism_directory = job_context["base_file_path"].split("/")[-1]
```

Comment:
I added a helper a while ago to get the filename from a path https://github.com/AlexsLemonade/refinebio/blob/dev/common/data_refinery_common/utils.py#L548-L550. Is it helpful here?

Reply:
It's not actually the filename that this is going for.... It's the directory name. I think this is a bit confusing but I think your function idea will help.